### ISSUE TITLE
Bug with `--data-var` not correctly being parsed

### ISSUE THREAD

**Issue by onedr0p** [mirkolenz/makejinja#91]
**Describe the bug**

Hi 👋🏼 

I am trying to override vars in a `--data-var` arg but I am getting an error, I am probably do it wrong but I have tried a whole bunch of different values for `--data-var`

### command

```
makejinja --input test.yaml.j2 --output test.yaml --data config.yaml --data-var "test=true" --jinja-suffix ".j2"
```

### output

```
Traceback (most recent call last):
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/bin/makejinja", line 8, in <module>
    sys.exit(makejinja_cli())
             ^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/rich_click/rich_command.py", line 126, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/typed_settings/cli_click.py", line 247, in new_func
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/makejinja/cli.py", line 36, in makejinja_cli
    makejinja(config)
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/makejinja/app.py", line 29, in makejinja
    data = load_data(config)
           ^^^^^^^^^^^^^^^^^
  File "/Users/devin/Code/repos/onedr0p/flux-template-cluster/.venv/lib/python3.11/site-packages/makejinja/app.py", line 283, in load_data
    for key, value in config.data_vars:
        ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
```

**Comment by mirkolenz** [comment]
Hi, thanks for opening this issue. It was caused by a bug in `makejinja`: `config.data_vars` is a dictionary, so one needs to iterate over `config.data_vars.items()` to access its keys and values.

I just published a fix in [v2.3.3](https://github.com/mirkolenz/makejinja/releases/tag/v2.3.3) together with a test case for `--data-var` to prevent that from happening again. If it still does not work with the new version feel free to reopen the issue.

**Comment by onedr0p** [comment]
Perfect, looks like it working. Thanks for the quick fix!
